evalml/tests/conftest.py

```diff
@@ -334,11 +334,36 @@ def _get_X_y(
         freq="D",
         problem_type="time series regression",
         match_indices=True,
+        n_series=1,
     ):
-        X = pd.DataFrame(index=[i + 1 for i in range(50)])
         dates = pd.date_range("1/1/21", periods=50, freq=freq)
-        feature = pd.Series([1, 4, 2] * 10 + [3, 1] * 10, index=X.index)
-        y = pd.Series([1, 2, 3, 4, 5, 6, 5, 4, 3, 2] * 5)
+        if n_series > 1:
```

Comment:
What's the purpose of this? It looks very similar to the multiseries fixtures that already exist, and it seems like there's a lot of repeated code. This function is already a beast to try to understand - if this separate fixture is necessary, can we leverage the other one to get this code more readable/less redundant?

Reply:
I think we should keep a separate fixture since this one makes generating synthetic datasets that have certain attributes (such as keeping certain indicies, etc.) easier. I would be open to splitting it into a separate `ts_mutiseries_data()` fixture if that helps with code complexity? 

Re: the existing `multiseries_ts_data_stacked`/`multiseries_ts_data_unstacked`, I think they serve separate uses so all three fixtures can exist.

Reply:
I think splitting is a great idea, keeping all three fixtures is fine by me